vocabularies/sample-relationship.ttl

```diff
@@ -53,15 +51,9 @@ sosa:hasOriginalSample
     skos:prefLabel "has original sample"@en ;
     sdo:domainIncludes sosa:Sample ;
     sdo:rangeIncludes sosa:Sample ;
-    skos:definition "A Link to the original sample that is related to the context sample through a chain of isSampleOf relations."@en ;
+    skos:definition "A link from the subject sample to the original sample of interest through a chain of isSampleOf relations."@en ;

```

Comment:
Could we mention "transitive" here - it is a term we are wanting people to embrace.

Reply:
```suggestion
    skos:definition "A transitive link from the subject sample to the original sample of interest through a chain of isSampleOf relations between multiple successive samplings of samples"@en ;

```

Too convoluted? @DavidCrosswellGSQ